javascript/auth-code-client-side/README.md

```diff
@@ -0,0 +1,51 @@
+# Authorization Code Sample
+
+This is a fully client-side implementation of the OAuth 2.0
+Authorization Code Grant. It's meant to be a reference implementation of
+using the Getty API with users using their own Getty or iStock
+credentials.
+
+This form of Authorization Code does not use a client secret. It does
+make use of the [PKCE](https://datatracker.ietf.org/doc/html/rfc7636)
+extension to aid in keeping the access token secure from any malicious
+scripts or apps on the user's device.
+
+This project does not rely on _any_ third-party libraries or packages.
+
+## Running locally
+
+After cloning this repository, the simplest way to test it out is by
+using [Docker](https://docs.docker.com/get-docker/).
```

Comment:
This says to use Docker and then doesn't tell you what to use it for.  Can we just provide a simple `docker run ...` command?

Reply:
There's a Makefile with the Docker command. Just type `make` and it will start the container.

Reply:
Oh, I see it's in your `Makefile`.  Should we assume `make` is present?

Reply:
How about just offer the Docker commands verbatim in the README:

- Start `docker run -d -v ${PWD}:/usr/share/nginx/html --name nginx-authcode --rm -p 8080:80 nginx:mainline-alpine`
- Stop `docker stop nginx-authcode`

Reply:
Added some info to the README